npm/create-cypress-tests/package.json

```diff
@@ -27,14 +27,16 @@
     "glob": "^7.1.6",
     "inquirer": "7.3.3",
     "ora": "^5.1.0",
-    "recast": "0.20.4"
+    "recast": "0.20.4",
+    "semver": "7.3.7"
   },
   "devDependencies": {
     "@types/babel__core": "^7.1.2",
     "@types/inquirer": "7.3.1",
     "@types/mock-fs": "4.10.0",
     "@types/node": "14.14.31",
     "@types/ora": "^3.2.0",
+    "@types/semver": "7.3.9",
```

Comment:
these types are used in this package but apparently were only available due to the miracle of hoisting, and by virtue of design-system including them